examples/utils/example-utils/src/migrationInterfaces/sameContainerMigrationTool.ts

```diff
@@ -27,7 +28,7 @@ export type SameContainerMigrationState =
 	| "collaborating"
 	| "proposingMigration"
 	| "stoppingCollaboration"
-	| "generatingV1Summary"
+	| "loadingV1PausedContainer"
 	| "uploadingV1Summary"
 	| "submittingV1Summary"
```

Comment:
Remove the other two v1 summary states too.

I don't think the MigrationTool needs to participate in loading the exporter client - see comment below for more thoughts on that.  So maybe no new states needed here.